eslint/babel-eslint-parser/src/client.cjs

```diff
@@ -60,7 +60,7 @@ exports.WorkerClient = class WorkerClient extends Client {
     { env: WorkerClient.#worker_threads.SHARE_ENV },
   );
 
-  #signal = new Int32Array(new SharedArrayBuffer(4));
+  #signal = new Int32Array(new SharedArrayBuffer(8));
```

Comment:
Why do we need doubling the byte length?

Reply:
```js
        if (resp) break;
        Atomics.wait(this.#signal, 1, 0, 30);
```
For convenience, I increased the byte length for sleep.